src/core/geometry/qgsgeos.h

```diff
@@ -297,7 +297,7 @@ class CORE_EXPORT QgsGeos: public QgsGeometryEngine
         QString *errorMsg = nullptr ) const;
 
     /**
-     * Returns the maximum inscribed circle.
+     * Returns the maximum inscribed circle radius.
```

Comment:
```suggestion
     * Returns the maximum inscribed circle.
```

it returns a geometry -- not a numeric value